default.nix

```diff
@@ -123,7 +123,7 @@ rec {
   inherit (reflex-platform) nixpkgs pinBuildInputs;
   path = reflex-platform.filterGit ./.;
   obelisk = ghcObelisk;
-  command = pkgs.runCommand "ob" { nativeBuildInputs = [pkgs.makeWrapper]; } ''
+  command = pkgs.runCommand "obelisk-command" { nativeBuildInputs = [pkgs.makeWrapper]; } ''
```

Comment:
Can't you get the name from the underlying derivation?

Reply:
You probably can, but I think pinning it explicitly might be even better, for the very reason that it matters.

Reply:
I think it would make sense for the wrapper to not be a name-changing thing.  If we want to ensure that the name doesn't change, I think it would be reasonable to add a test for that.